mypy/main.py

```diff
@@ -203,6 +204,16 @@ def add_invertible_flag(flag: str,
             strict_flag_names.append(flag)
             strict_flag_assignments.append((dest, not default))
 
+    def disallow_any_argument_type(raw_options: str) -> List[str]:
+        current_options = raw_options.split(',')
```

Comment:
I'm not sure this is named quite right.  "current_options" implies that the options will be changing over time, which doesn't really happen.  Maybe consider just "options"?